.travis.yml

```diff
@@ -9,6 +9,10 @@ env:
           testflags="--test-type integ --pytest-args='-x'"
           use_codecov=true
           use_coveralls=false
+        - CONTAINER_IMAGE=nmstate/centos8-nmstate-dev
```

Comment:
How about running two containers in the same job instead of adding another job?

Reply:
You assume multiple CPU/s to carry the task which may not be a valid assumption.
It will also mess up the logs, I have bad experience from the oVirt project with this.

Reply:
According to Travis, there are two cores available. Also needing more cores assumes that the tests are CPU bound but there is also a lot of sleeping happening which makes the CPU idle and would allow to run something else in parallel.

Reply:
Ack for the CPU/s, but AFAIK the logging of running two containers in parallel has no known solution.

Reply:
We could use pytest-xdist to have one pytest command run on both containers: https://pypi.org/project/pytest-xdist/

Reply:
This will require a lot more investigation and work.
At the moment I do not know how it will fit exactly.